Description: install new header files and fix name of installed include
 directory
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ndpi/src/include/Makefile.am
===================================================================
--- ndpi.orig/src/include/Makefile.am
+++ ndpi/src/include/Makefile.am
@@ -1,9 +1,14 @@
 
-library_includedir=$(includedir)/libndpi-1.4/libndpi
+library_includedir=$(includedir)/libndpi-@VERSION@/libndpi
 
-library_include_HEADERS = ndpi_api.h \
+library_include_HEADERS = \
+		  linux_compat.h \
+		  ndpi_api.h \
 		  ndpi_debug_functions.h \
 		  ndpi_define.h \
 		  ndpi_macros.h \
+		  ndpi_main.h \
+		  ndpi_protocol_history.h \
 		  ndpi_protocols_osdpi.h \
-		  ndpi_public_functions.h
+		  ndpi_public_functions.h \
+		  ndpi_structs.h
